### ISSUE TITLE
stream_commands_test.py tests fail on Redis v5.0.3

### ISSUE THREAD

**Issue by gyermolenko** [aio-libs/aioredis-py#529]
Tests from `stream_commands_test.py` fail.
redis v5.0.3
macos

```
============================================================================ FAILURES =============================================================================
______________________________________________________________ test_xread_group[redis_v5.0.3-single] ______________________________________________________________

redis = <Redis <RedisConnection [db:0]>>

    @pytest.mark.run_loop
    async def test_xread_group(redis):
        await redis.xadd('test_stream', {'a': 1})
        await redis.xgroup_create('test_stream', 'test_group', latest_id='0')

        messages = await redis.xread_group(
            'test_group', 'test_consumer', ['test_stream'],
            timeout=1000, latest_ids=[0]
        )
>       assert len(messages) == 1
E       assert 0 == 1
E        +  where 0 = len([])

tests/stream_commands_test.py:336: AssertionError
```
```
===================================================================== short test summary info =====================================================================
FAIL tests/stream_commands_test.py::test_xread_group[redis_v5.0.3-single]
FAIL tests/stream_commands_test.py::test_xread_group[redis_v5.0.3-pool]
FAIL tests/stream_commands_test.py::test_xack_and_xpending[redis_v5.0.3-single]
FAIL tests/stream_commands_test.py::test_xack_and_xpending[redis_v5.0.3-pool]
FAIL tests/stream_commands_test.py::test_xpending_get_messages[redis_v5.0.3-single]
FAIL tests/stream_commands_test.py::test_xpending_get_messages[redis_v5.0.3-pool]
FAIL tests/stream_commands_test.py::test_xclaim_simple[redis_v5.0.3-single]
FAIL tests/stream_commands_test.py::test_xclaim_simple[redis_v5.0.3-pool]
FAIL tests/stream_commands_test.py::test_xclaim_min_idle_time_includes_messages[redis_v5.0.3-single]
FAIL tests/stream_commands_test.py::test_xclaim_min_idle_time_includes_messages[redis_v5.0.3-pool]
============================================================== 10 failed, 36 passed in 2.15 seconds
```

- Not reproduced with v5.0.1 (from http://download.redis.io/releases/)
```
tests/stream_commands_test.py ..............................................                                                                                [100%]

==================================================================== 46 passed in 1.93 seconds
```
- can't be tested with v4.* since it doesn't have streams

**Comment by gyermolenko** [comment]
well, I reproduced the issue in `redis-cli` so it is not about aioredis or its tests. 
Here are actions from the test
```
> xadd test_stream * a 1
> xgroup create test_stream test_group 0
> xreadgroup group test_group test_consumer streams test_stream 0
```

Here are my console outputs:
**5.0.1 local server**
```

$ redis-cli
127.0.0.1:6379> keys *
(empty list or set)
127.0.0.1:6379> xadd test_stream * a 1
"1548840128175-0"
127.0.0.1:6379> xgroup create test_stream test_group 0
OK
127.0.0.1:6379> xreadgroup group test_group test_consumer streams test_stream 0
1) 1) "test_stream"
   2) 1) 1) "1548840128175-0"
         2) 1) "a"
            2) "1"

127.0.0.1:6379> xpending test_stream test_group
1) (integer) 1
2) "1548840128175-0"
3) "1548840128175-0"
4) 1) 1) "test_consumer"
      2) "1"

```

**5.0.3** docker (reproduced with redis v5.0.3 from homebew)
```
$ redis-cli -u redis://localhost:6333/0
localhost:6333> keys *
(empty list or set)
localhost:6333> xadd test_stream * a 1
"1548840749309-0"
localhost:6333> xgroup create test_stream test_group 0
OK
localhost:6333> xreadgroup group test_group test_consumer streams test_stream 0
1) 1) "test_stream"
   2) (empty list or set)

localhost:6333> xpending test_stream test_group
1) (integer) 0
2) (nil)
3) (nil)
4) (nil)
```

**Comment by decaz** [comment]
I guess it's because tests are adopted for the 5.0.1 version but there is streams bug was fixed within 5.0.2 version (https://github.com/antirez/redis/blob/5.0/00-RELEASENOTES#L222-L236) so now tests will fail if version >= 5.0.2.

FYI @popravich

**Comment by gyermolenko** [comment]
@decaz hi. 
I experienced the defect with 5.0.3, not with 5.0.1. With redis itself (wo aioredis).
If someone can't reproduce behavior described above in redis-cli - I need to investigate further. 
Otherwise - all we can do is to wait for redis with the fix (maybe 5.0.4)

**Comment by gyermolenko** [comment]
reproduced with redis:5.0.5 from (in docker)
